### DISCUSSION TITLE
Your setup?

### DISCUSSION THREAD

**Discussion by Ghr00t** [streamdeck-linux-gui/streamdeck-linux-gui#60] [Show and tell]
I'll post mine to go first.  It's not much, and i'm looking for other ideas.
I use amixer for volume controls, and playerctl for media controls.  Icon pack is free from Nerd or Die.

![image](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/87456599/647ed84b-56dc-41b5-9f32-6a49f586978d)

**Comment by Data-007** [comment]
![My Streamdeck](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/53057579/b9b161da-1065-47b4-afac-5490b24faa19)

**Comment by oparada1988** [comment]
I make my own icons using inkscape :-)

![Screenshot from 2023-08-28 18-15-34](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/bf93c94e-bb86-46a9-8c2c-beefec8179dd)

![Screenshot from 2023-08-28 18-15-40](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/3026633f-e51e-425d-ab0b-16704b45a8a8)

**Comment by rkmax** [comment]
My "home" page include many animations

https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/395707/92c28d60-7498-41fd-ac17-08f13f4d07c4

**Comment by Danie10** [comment]
OK here is mine - basically some OBS Studio shortcuts, Zooms calls, general browser zoom in/out, and some keys to toggle compositing on Plasma, running Manjaro updates, and quick access to two Obsidian reference notes. The Snowrunner page branches off into another page just for crane controls (but that is waiting for the repeat keypress on hold, as you hold a key in to raise/lower the crane, rotate etc). 

https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/1153726/41fe8396-a29c-469f-b673-44113ac9e00e

**Comment by ClaudiusMinimus** [comment]
Could any of you upload your `.streamdeck_ui.json` -- _removing any personal information_ -- so other noobs like me can see the json. It would help to see working examples. For example, I would love to see examples of [Sending key presses to a specific open Window with xdotool](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/56)
Thank you!

**Comment by Danie10** [comment]
@ClaudiusMinimus here is an excerpt of one of mine, but I'd suggest you plug this type of command in using the GUI anyway. It is pretty well much the same as I put in my post about this - you paste it in:

```
"2": {
                        "state": 0,
                        "states": {
                            "0": {
                                "text": "",
                                "icon": "/home/danie/Pictures/Icons/IconPacks/Symbols/icons/2b06.svg",
                                "keys": "",
                                "write": "",
                                "command": "xdotool search --name '^OBS [0-9]+\\.[0-9]+\\.[0-9]' key ctrl+KP_Up",
                                "brightness_change": 0,
                                "switch_page": 0,
                                "switch_state": 0,
                                "text_vertical_align": "",
                                "text_horizontal_align": "",
                                "font": "",
                                "font_color": "",
                                "font_size": 0,
                                "background_color": ""
                            }
                        }
                    },
```

**Reply by ClaudiusMinimus** [reply]
Thank you - this is perfect! I was most interested in the command property arguments. Is there a repository of example shell scrips? I've been looking, but have not found any yet.
Thank you!

**Reply by Danie10** [reply]
I don't think that would be easily possible as every button is dependent on the specific key as well as even window name of the application. What works for me won't necessarily work for you if I'm using a beta version of the app etc. There are also other ways of doing a similar thing with other commands or sometimes just keytrokes. It was why I made the other post about the keypresses, to try to explain what could be possible to do, and what the format looks like to use.

**Reply by ClaudiusMinimus** [reply]
I agree with you and that is a very important point for any examples. However, being able to see how others have set up their Stream Decks would be very helpful, especially to those new to scripting. [Inline commenting](https://www.freecodecamp.org/news/comments-in-json/#:~:text=Why%20JSON%20Does%20Not%20Support,be%20quickly%20and%20efficiently%20processed.) would also be useful by explaining the purpose of the command or logic syntax presented. Comments could also show where window names or other key variables are situation dependent and must be set according to the user's hardware/software configuration.
